docs/reference/oop/file.md

```diff
@@ -0,0 +1,6 @@
+::: synapseclient.models.File
```

Comment:
Not sure why all of these md files are in my PR, I did not create them all. Were they deleted or moved in `develop` or something?

Reply:
I had them in the PoC originally - But removed them in favor of putting these into a single file:
https://github.com/Sage-Bionetworks/synapsePythonClient/blob/5f2e2b742d72c9aaf04ce3446d83744426d355dc/docs/reference/oop/models.md

Reply:
OK I will remove the individual ones here.